Source/p_map.c

```diff
@@ -40,6 +40,8 @@
 #include "m_random.h"
 #include "m_bbox.h"
 
+extern int V_BloodColor(int blood);
```

Comment:
Rule of thumb: If you have to declare an extern function twice, put it into a header. :wink: